tests/kas_messaging.spec.ts

```diff
@@ -94,6 +94,19 @@ test.beforeEach(async ({ page }) => {
   expect(producerResponse === true).toBeTruthy();
 });
 
+test.afterAll(async ({ page }) => {
+  await navigateToKafkaList(page);
+
+  if ((await page.getByText(testInstanceName).count()) !== 0) {
```

Comment:
it would be better to just execute the action in a `try` block(i.e. with no `if`), so that PW will fully leverage autowait.

Reply:
I think this will fail faster?. In case PW will try to click on something and the element won't be there, we will wait 30 sec till failure. This could add some additional execution time which is not needed.

Reply:
On the contrary, if an element is not "stable"(e.g. React is taking a few milliseconds to get into the correct state or console-dot is triggering an async refresh) we already observed failures...
ok, not fussy about this anyhow, we can start with one approach and eventually switch to the other in the case is not effective 👍

Reply:
Yesterday we hit issue with tests based on https://issues.redhat.com/browse/MGDSTRM-9359 (fix https://github.com/redhat-developer/consoledot-e2e/pull/76) so in that case, try/catch makes much more sense. However, we should adjust the timeouts I thin having 30s timeout for waiting some element that will actually never appear (expected behavior) is not a good approach. Overall I think we should choose checks based on expected behavior.

Reply:
rewritten to try/catch block